Cargo.toml

```diff
@@ -457,3 +457,11 @@ incremental = false
 debug-assertions = false
 overflow-checks = false
 opt-level = 's'
+
+[profile.optimized]
+inherits = "release"
+codegen-units = 1
+incremental = false
+lto = true
+panic = "abort"
+strip = "none"
```

Comment:
Why disable debuginfo stripping when you don't enable debuginfo anyway?

Reply:
This strips more, though (symbols as well). We really need all these settings to close the gap to a build without wasmtime (we tested this).

Reply:
`strip = "none"` strips neither debuginfo, nor symbols.

Reply:
My mistake; I was reading this wrong (my own tests were with `true`).

Reply:
I benchmarked this and it seems strip does not seem to make difference. I'll remove it.